lib/Eclair/LLVM/BTree.hs

```diff
@@ -35,7 +35,7 @@ instance Pretty Meta where
   pretty meta =
     "num_columns=" <> pretty (numColumns meta) <> comma <+>
     -- TODO: use "withCommas"
-    "index=" <> brackets (F.fold $ intersperse comma $ map pretty (index meta)) <> comma <+>
+    "index=" <> brackets (Prelude.fold $ intersperse comma $ map pretty (index meta)) <> comma <+>
```

Comment:
Should I use `cata` or reverse it?

Reply:
Ah, no this is actually a place where `cata` would be more cumbersome.

The annoying thing about recursion-schemes is that it re-exports `cata` as `fold`, but it's different than the `fold` from `Data.Foldable`. This is the correct function.